packages/x-components/src/x-modules/search-box/components/clear-search-input.vue

```diff
@@ -124,11 +126,10 @@ In this example, the `UserPressedClearSearchBoxButton` event is implemented, tri
 
 _Click the Clear button to try it out!_
 
-```vue
+```vue live
 <template>
```

Comment:
`<template>` can only have one root child. Wrap the components in a div for the code to work